mapit_gb/data/authorities.json

```diff
@@ -0,0 +1,2217 @@
+{
+  "buckinghamshire": {
+    "name": "Buckinghamshire County Council",
+    "ons": "11",
```

Comment:
We aren't using the SNACs or names from this file, so maybe we should remove them in a separate commit. We know we want to [add more accurate names into MapIt](https://trello.com/c/ntfBCKeJ/277-get-correct-la-names-into-mapit) for these areas so preserving this data structure has some value (instead of switching to a simpler structure like `gss_code: slug`) but these names aren't up-to-date at the moment and they aren't the ones in our database so it's probably less confusing to remove them (or change all the `name` values to `null`?).